### DISCUSSION TITLE
tagstack macro mysteriously disappeared

### DISCUSSION THREAD

**Discussion by tksoh** [eteran/nedit-ng#244] [General]
Today, I wanted to test out my tagstack macros with the latest master branch. When I tried to run 'push tagstack' through the Macro menu, I only saw the 'pop tagstack' menu, while 'push tagstack' is no where to be found. I have since restored the 'push tagstack' manually, even though 'pop' still missing the raise_window() function.

I don't know what happened, nor how to recreate the problem, but thought I should keep a record for this strange incident.

**Comment by eteran** [comment]
That is definitely strange! please let me know if it happens again.

**Comment by eteran** [comment]
@tksoh Sorry, that was a bug I introduced, it should be fixed now. Another user reported it at the end of this issue here:

https://github.com/eteran/nedit-ng/issues/238
